backend/src/Squidex.Domain.Apps.Entities/Apps/Templates/Builders/ArrayFieldBuilder.cs

```diff
@@ -0,0 +1,117 @@
+﻿// ==========================================================================
+//  Squidex Headless CMS
+// ==========================================================================
+//  Copyright (c) Squidex UG (haftungsbeschraenkt)
+//  All rights reserved. Licensed under the MIT license.
+// ==========================================================================
+
+using System;
+using System.Linq;
+using Squidex.Domain.Apps.Core.Schemas;
+using Squidex.Domain.Apps.Entities.Schemas.Commands;
+using Squidex.Text;
+
+namespace Squidex.Domain.Apps.Entities.Apps.Templates.Builders
+{
+    public class ArrayFieldBuilder : FieldBuilder
+    {
+        protected new UpsertSchemaField field
+        {
+            get => base.field as UpsertSchemaField;
+            init => base.field = value;
+        }
+
+        public ArrayFieldBuilder(UpsertSchemaField field, CreateSchema schema)
+            : base(field, schema)
+        {
+        }
+
+        public new ArrayFieldBuilder Label(string? label) => base.Label(label) as ArrayFieldBuilder;
```

Comment:
Two things:

1. Can this not be solved with a generic base class: `ArrayFieldBuilder : FieldBuilder<ArrayFieldBuilder>`
2. Please do not use expression members, not a fan.